content/sensu-core/2.0/guides/install-check-executables-with-assets.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "How to install Check Executables using Assets"
+linkTitle: "Install Check Executables Using Assets"
```

Comment:
We might want to use a shorter title:

![screen shot 2018-04-06 at 1 45 33 pm](https://user-images.githubusercontent.com/4105580/38436082-acaba060-39a1-11e8-975d-ec5c4d242dd4.png)

Reply:
Ah yeah, I will think of something shorter!

Reply:
@palourde updated!